### PR TITLE
Deprecate wink.

### PR DESCRIPTION
**Pull request by janiversen**
## Proposed change

Deprecate wink

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/54496

- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/wink.markdown [hunk 1]
```diff
@@ -33,6 +33,25 @@ ha_platforms:
 ---
 
 [Wink](https://www.wink.com/) is a home automation hub that can control a whole wide range of devices on the market. Or, as they say in their own words:
+<div class="note warning">
+ The Wink integration has been marked deprecated and is pending removal in Home Assistant Core 2021.12.
+ It is considered impossible to obtain the needed API keys to use this integration.
+ </div>
+
+ <div class="note warning">
+ The Lyft integration has been marked deprecated and is pending removal in Home Assistant Core 2021.10.
```
Comment (frenck) [inline source/_integrations/wink.markdown]: ?

Comment (janiversen) [reply source/_integrations/wink.markdown]: ups, sorry I did this on my ipad.

File: source/_integrations/wink.markdown [hunk 2]
```diff
@@ -33,6 +33,10 @@ ha_platforms:
 ---
 
 [Wink](https://www.wink.com/) is a home automation hub that can control a whole wide range of devices on the market. Or, as they say in their own words:
+<div class="note warning">
+ The Wink integration has been marked deprecated and is pending removal in Home Assistant Core 2021.12.
+ It is considered impossible to obtain the needed API keys to use this integration.
```
Comment (frenck) [inline source/_integrations/wink.markdown]:
```suggestion
 It is considered impossible to obtain the needed API keys to use this integration. The developer portal has been taken down and support has confirmed to their users they won't be allowing new applications.
```

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by janiversen** [comment]
Parent is closed

**Comment by frenck** [comment]
Updated parent

**Review by cgarwood — APPROVED** [review]

**Pull request merged**